src/domains/dashboard/components/node-view/render-submenu-name.tsx

```diff
@@ -124,6 +124,9 @@ export const renderSubmenuName = ({
                 decimalDigits: netdataDashboard.contextDecimalDigits(
                   chart.context, -1,
                 ),
+                selectedDimensions: Object.values(
+                  chartsMetadata.charts[chart.id]?.dimensions || {},
```

Comment:
@VLegakis i understand that `chartMetadata` is modified in the cloud and sent to dashboard. I see few problems with that. First is that Dashboard expects that metadata to be the same as the one we fetch from BE. It looks like chartsMetadata were changed only to omit props-drilling, which IMO shouldn't happen. If we have problem with props drilling, we should perhaps go around it with putting something to redux state, or create some other configuration-like object, which is later mapped to attributes in Dashboard?

Also, the `dimensions` in chartsMetadata will not be used in the future, and it's even possible that we'll remove that support from the Dashboard. That's because the dimensions can change during chart updates (can be added dynamically by result of data calls), and we fetch metadata only once, on init time. We use `dimensions` from metadata currently only for colors, but it's actually wrong and there's a bug related to this.

Reply:
Ok understood. Maybe got the `attributesOveride` way then. Lets discuss this after the demo

Reply:
Done